easybuild/easyconfigs/w/WSClean/WSClean-3.4-foss-2023b.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'WSClean'
+version = '3.4'
+
+homepage = 'https://wsclean.readthedocs.io/'
+description = """WSClean (w-stacking clean) is a fast generic widefield imager.
+It implements several gridding algorithms and offers fully-automated multi-scale
+multi-frequency deconvolution."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+sources = [
+    {
+        'source_urls': ['https://gitlab.com/aroffringa/%(namelower)s/-/package_files/97237455/'],
+        'filename': '%(namelower)s-v%(version)s.tar.bz2',
+        'download_filename': 'download'
+    },
+]
+checksums = ['b43d8ca490ccf34dd22aae6c5ca88a5dcb3cff0526835d3f97fa6d239745e641']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+]
+dependencies = [
+    ('casacore', '3.5.0'),
+    ('EveryBeam', '0.5.2'),
+    ('Boost', '1.83.0'),
+    ('CFITSIO', '4.3.1'),
+    ('GSL', '2.7'),
+    ('HDF5', '1.14.3'),
+    ('Python', '3.11.5'),
+]
+
+# Without this, the wrong (system) python is picked up by cmake
+# in cases where the system python is newer than the one used in these EasyConfigs
+# See PR # 19119
+configopts = "-DPython3_EXECUTABLE=$EBROOTPYTHON/bin/python "
```

Comment:
This should now be fixed in the CMakeMake EasyBlock by https://github.com/easybuilders/easybuild-easyblocks/pull/3088 . Please remove this configopts addition and check it still works (we can check when building for EESSI, where you found the issue previously....)